packages/ckeditor5-engine/src/conversion/downcastdispatcher.js

```diff
@@ -16,16 +16,16 @@ import { extend } from 'lodash-es';
 /**
  * `DowncastDispatcher` is a central point of downcasting (conversion from model to view), which is a process of reacting to changes
  * in the model and firing a set of events. Callbacks listening to those events are called converters. Those
- * converters role is to convert the model changes to changes in view (for example, adding view nodes or
+ * converters' role is to convert the model changes to changes in view (for example, adding view nodes or
  * changing attributes on view elements).
  *
  * During conversion process, `DowncastDispatcher` fires events, basing on state of the model and prepares
  * data for those events. It is important to understand that those events are connected with changes done on model,
- * for example: "node has been inserted" or "attribute has changed". This is in a contrary to upcasting (view to model conversion),
+ * for example: "node has been inserted" or "attribute has changed". This is in contrary to upcasting (view to model conversion),
  * where we convert view state (view nodes) to a model tree.
  *
  * The events are prepared basing on a diff created by {@link module:engine/model/differ~Differ Differ}, which buffers them
- * and then passes to `DowncastDispatcher` as a diff between old model state and new model state.
+ * and then passes to `DowncastDispatcher` as a diff between the old model state and the new model state.
  *
  * Note, that because changes are converted there is a need to have a mapping between model structure and view structure.
```

Comment:
```suggestion
 * Note that because the changes are converted, there is a need to have a mapping between the model structure and the view structure.
```